inc/saiacl.h

```diff
@@ -227,17 +241,36 @@ typedef enum _sai_acl_table_group_attr_t
     SAI_ACL_TABLE_GROUP_ATTR_START,
 
     /**
-     * @brief Dummy attribute
+     * @brief ACL stage
+     *
+     * @type sai_acl_stage_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_ACL_STAGE = SAI_ACL_TABLE_GROUP_ATTR_START,
```

Comment:
can we add SAI_ACL_TABLE_GROUP_ATTR_TYPE attribute?

The group type can be an enum 
{

acl_table_group_type_parallel 

acl_table_group_type_sequential
}

In paralled, all acl tables in the group are looked up in parallel, and non conflicting actions are taken from multiple matched acl entries.

In sequential mode, priority is defined between the acl tables, and only one acl entry is matched.